### DISCUSSION TITLE
Submit Radioss job from arbitrary directory/folder

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1406] [Users]
Hello everyone,

I am new to OpenRadioss adn still learning. Could you please help me on the questions below?

1. I have my first model successfully runs on my windows machine, but it seems everything has to be in the same folder where I have my Openradioss installed? How can I submit Radioss job from arbitrary directory/folder, then let the outcomes/results saved in the folder where I submit my jobs?
2. How to require the Ann and Txx outputs? Then I can post-process my results in Paraview.
3. Any other post-processing tools recommended?

Thanks for any helps/comments

Best
Lee

**Comment by li2qn** [comment]
any help? thanks

**Reply by MarianBulla** [reply]
Hello Lee,
and congratulations to get OpenRadioss running on your machine.
Regarding your questions:
1. You don't need to run your models in the same folder, where your executables are. You just need to set the path variable which points to your executables. Just create a batchfile (e.g.: "OpenRad.bat") setting all necessary environmental variables. Here it is an example, which works on my laptop: 
![image](https://user-images.githubusercontent.com/105412502/234773809-27feb3a4-867d-42d5-b863-9671f476c28f.png)
![image](https://user-images.githubusercontent.com/105412502/234774945-f8b03343-dbe1-4926-aa86-51e0d571daa7.png)

Another way is to use the nice GUI, created by Paul (see in the discussion forum for further details). This starts a small graphical user interface, which makes it even more simpler to start you maodel:
![image](https://user-images.githubusercontent.com/105412502/234775164-0b357827-6435-4af7-b073-fadb24e709d9.png)

2. Within your engine file (name_0001.rad file) or the engine block in the model file (name_0000.rad file), you just add these requests for your desired outout. Please, review the examples and the documentation for further details.

3. For postprocessing tools, you can use HyperWorks (HyperView and HyperGraph) directly. If you are within an university, you can apply for a student edition. If you you want to got with ParaView, you need to convert the animation files and the T01 file into VTK and *csv files. The necessary tools are within the TOOLS folder, also including a description how to do it.

I hope this helps you ?

Thanks and best regards,
Marian
